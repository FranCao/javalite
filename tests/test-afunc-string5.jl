int main() {
    string s;
    int l;
    s = "he" + "y";
    print(s);
}