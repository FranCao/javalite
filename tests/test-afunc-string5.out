1
6
-1
Hello World!!FooFooDooDooBar
Hello FooFooDooDooBar!!!!!